ddd-koans/Value Object/Immutablity/task.html

```diff
@@ -0,0 +1,37 @@
+<html>
+<h3>
+    Immutability is a one of characteristics of <b>Value Object</b>. <br/>
+    It means that once <b>value object</b> is created, it can never be changed. So the state is unchangeable.
+</h3>
+<p>
+    In this task and at least few next you will develop a <b>TimeSignature</b> class. <br/>
+    It could serve as a great example of a <b>value object</b>. <br/>
+    So here we go with a reminder of what time signatures are:
+</p>
+<h4>
+    Time signatures are symbols which describe repeating beat rhythm in a piece of music. <br/>
+    It tells us the total length of all symbols in a bar. <br/>
+    The upper value is called numerator and lower denominator.
+    <br/>
+    <br/>
+    <img src="3over4.png">
+</h4>
+<p>
+<div>
+    As the description states <b>TimeSignature</b> must contain only two properties:
+    <ul>
+        <li>nominator,</li>
+        <li>denominator,</li>
+    </ul>
+    both of them should be of type <b>Int</b> and declared using <b>val</b> not
```

Comment:
I wonder if it wouldn't too easy giving such explanation about `val`. Maybe as a hint? 🤷‍♂️

Reply:
Yeah, that was too easy. That way it would be better.